Makefile

```diff
@@ -74,8 +74,8 @@ do-show-commands:
 	@echo "    make preview-docs               Run a live preview of the documentation."
 	@echo "\nDevelopment:"
 	@echo "    make dev-server                 Run the development server."
-	@echo "    make dev-server-slave           Run the development slave server, listening to a master."
-	@echo "    make dev-client                 Build, run and watch the development dashboard."
+	@echo "    make dev-module-client          Run the development slave server, listening to a master."
```

Comment:
Please update the description of this make command.

Reply:
Fixed.